testcafe/helpers.js

```diff
@@ -51,15 +67,13 @@ export async function retryUntilResults(operation, target_results, limit = 100)
     return new Promise((...args) => attempt(0, ...args))
 }
 
-export async function queryAPI() {
-    const response = await fetch('http://localhost:8000/api/event', {
+export async function queryAPI(testSessionId) {
+    const url = `${POSTHOG_API_HOST}/api/projects/${POSTHOG_API_PROJECT}/events?properties=[{"key":"testSessionId","value":["${testSessionId}"],"operator":"exact","type":"event"}]`
+    const response = await fetch(url, {
         headers: HEADERS,
     })
 
-    const { results } = JSON.parse(await response.text())
+    const data = await response.text()
```

Comment:
Should we explode if `!response.ok` so future errors are clearer?

Reply:
That's a good idea. I'll add it in